src/controllers/userController.js

```diff
@@ -1,29 +1,33 @@
-const userService = require('../services/userService');
-const logger = require('../helpers/logger');
+const userService = require('../services/userService')
+const logger = require('../helpers/logger')
 
 const createUser = async (req, res) => {
-    const { email } = req.body;
+    const { email } = req.body
 
-    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
     if (!email || !emailRegex.test(email)) {
```

Comment:
in this case you don't need to check if email is specified,  !emailRegex.test(email) condition will be enough to validate the email